include/constants/battle.h

```diff
@@ -617,9 +617,9 @@ typedef enum ControllerCommand {
     CONTROLLER_COMMAND_POKEMON_INPUT, // 15
     CONTROLLER_COMMAND_RUN_INPUT,
     CONTROLLER_COMMAND_SAFARI_THROW_BALL,
+    CONTROLLER_COMMAND_SAFARI_THROW_BAIT,
     CONTROLLER_COMMAND_SAFARI_THROW_MUD,
-    CONTROLLER_COMMAND_SAFARI_RUN,
-    CONTROLLER_COMMAND_SAFARI_WATCHING, // 20
+    CONTROLLER_COMMAND_SAFARI_RUN, // 20
```

Comment:
I think the matching issue is here (mud went from 18 -> 19 and the corresponding functions weren't renamed) so ensure that everything is in alignment properly. To make things easier, you can compare overlay 12 using asmdiff in tools/ and figure out exactly where and what the differences are.

Reply:
Got it, the functions were working correctly, the names just didn't match up, so moving these enums around screwed up the actual functionality.